National Library of Poland ISBN.js

```diff
@@ -0,0 +1,118 @@
+{
+	"translatorID": "aa7f310e-10d3-4209-91dc-88301e7070c6",
+	"label": "National Library of Poland ISBN",
+	"creator": "Maciej Nux Jaros",
+	"target": "",
+	"minVersion": "4.0",
+	"maxVersion": "",
+	"priority": 97,
```

Comment:
We'd want the priority lower -- I think this should probably run after the K10+ catalog or after the BnF

Reply:
Lower priority is higher number? Changed to the same as bnf (98)